docs/en/user/user-documentation.md

```diff
@@ -0,0 +1,95 @@
+* **Presented order** allows an editor to change the ordering of columns with more precision than dragging and dropping items in the CMS table. This also makes it easy to move items between pages.
+* The **Result conditions** allow for a "pre-filter" to be chosen to only show results matching (or not matching) a particular value.
+
+## Filters:
+The filters tab allows a CMS content editor to configure fields shown to a visitor that will allow them to search the results shown to them. One is always created by default after selecting a resource and saving the page. This default filter is a text type that searches all columns for matches on the visitor's search term.
+
+By default there are two types of filters available to a content editor to configure, text and dropdown. The text filter allows a user to enter text to filter results. The dropdown filter only allows a pre-defined value to be chosen.
+
+### Configuring a text filter:
+
+![Configuration options for a text filter](textFilterConfig.png)
+
+* A **Filter label** can be given to indicate what type of search the content editor configures it to do.
+* **Column sources** allow the filter to be configured to either search all columns, or if the checkbox is unchecked to search only a particular column or set of columns (multiple may be chosen).
+
+### Configuring the dropdown filter:
+
+This filter type has only one extra configuration option from those provided by the text filter; the ability to supply filter options for the user to choose from. There are two methods available to define options, however only the chosen method is used (the two cannot be combined).
+
+#### Options
+
+After choosing which columns the filter will apply to, the **Select from all options** area will update to provide a list of all possible unique values those columns contain. The content editor can then choose to disable any of these options by un-checking the corresponding checkbox.
+
+![Dropdown filter configuration](dropdownFilterConfig.png)
+
+#### Delimiter
+
+If these options are grouped, e.g. with the _Family Services Directory_ we can see a _`SERVICE_TARGET_AUDIENCES`_ column - any row for this column may contain multiple values separated by a comma, e.g. "Children,Youth"
+
+To get these values to display as both "Children" and "Youth" we can enter a comma into the **Delimiter** box; after clicking **Update** the list of options to will update to be separated appropriately.
+
+![Dropdown filter configuration using Delimiter](delimiterUse.png)
+
+#### Manual selection
+
+The **Manually add options** entry box for this allows values to be entered directly by the content editor, one option per line.
+
+E.g.
+
+    Children
+    Older People
+    Parents and Caregivers
+
+This option is available for those who find it easier to list out values they know in advance, rather than to enable or disable values individually.
```

Comment:
```suggestion
This option is available for cases where it might be easier to list out known values in advance, rather than to enable or disable provided values individually.
```